tests/unit/wer_report_test_wrapper.cpp

```diff
@@ -90,7 +92,7 @@ WerReportAddDump_test(
     _In_ WER_DUMP_TYPE dump_type,
     _In_opt_ WER_EXCEPTION_INFORMATION* exception_param,
     _In_opt_ WER_DUMP_CUSTOM_OPTIONS* dump_custom_options,
-    _In_ DWORD dwFlags)
+    _In_ unsigned long dwFlags)
```

Comment:
Normally (including in Windows API guidance) the rule is to not use `_In_` with pass-by-value types, only pointer types (pass by reference).  It's ok if it's required to match a Windows header file that doesn't follow the guidance, where compiler warnings/errors would result if we tried to follow the guidelines.  I'm guessing that may be the case here.